eng/common/scripts/Package-Properties.ps1

```diff
@@ -171,12 +171,12 @@ function Get-PkgPropsForEntireService ($serviceDirectoryPath)
 function Get-ArtifactListFromYml ($ciYmlPath)
 {
     $ProgressPreference = "SilentlyContinue"
-    if ((Get-PSRepository | ?{$_.Name -eq "PSGallery"}).Count -eq 0)
```

Comment:
@chidozieononiwu this is a couple more strict mode fixes I added to this PR that I hit while testing after rebasing my changes.